### ISSUE TITLE
OG bug: save crystals are affected by light point if both share the same space

### ISSUE THREAD

**Issue by aredfan** [LostArtefacts/TRX#1514]
**Regression since:** #1510 
**Saves:** [save_tr1_03.zip](https://github.com/user-attachments/files/17002059/save_tr1_03.zip)

________________________

The dev snapshot before 1510:

https://github.com/user-attachments/assets/453161c8-375c-490a-bc38-5426ab29469c

The dev snapshot after 1510:

https://github.com/user-attachments/assets/24b4888a-fd67-4751-be02-55c6cac783ff

It looks like there's a lighting issue within the reflection. I suspect this bug happens because there's a light point inside the save crystal. 

After going through all the OG levels on trview, I only found 2 save crystals with a light point inside.

Colosseum, room 0.
![Screenshot 1](https://github.com/user-attachments/assets/db4a7c10-6795-437a-8b40-65d293ba5faa)
The Cistern, room 16.
![Screenshot 2](https://github.com/user-attachments/assets/2a5d505e-2183-450e-ad78-30c7b720b01a)

**Comment by lahm86** [comment]
I think this is an OG bug? If you check in the current release you can see the lighting performs the same way on the flat crystal. So the current build is more in line with OG than the dev snapshot before 1510, I would say.
You get a similar sort of uneven lighting if you consume the crystal and just have Lara stand in the spot and turn around.
Data issue ultimately.

**Comment by rr-** [comment]
I agree with Lahm's analysis – it's a data issue with the light placement.

**Comment by lahm86** [comment]
We could inject a Y shift for the lights, but it would likely produce subtle differences on the room's overall lighting.

**Comment by Richard-L** [comment]
Tried reproducing it on PS1 (as to me that is "OG"). If it should help here's a clip. Looks normal.

https://github.com/user-attachments/assets/e131ef57-408e-489a-b3e0-1326156105c5

**Comment by aredfan** [comment]
I'm ok with shifting the light points if it can be done in a subtle and vanilla-friendly way. At least it's a rare bug, all things considered.

**Comment by rr-** [comment]
Ideally we should be able to match PS1 without changing the data.
